mount/mount_unix.go

```diff
@@ -37,35 +39,56 @@ func Unmount(target string) error {
 	}
 }
 
-// RecursiveUnmount unmounts the target and all mounts underneath, starting
-// with the deepest mount first. The argument does not have to be a mount
-// point itself.
-func RecursiveUnmount(target string) error {
-	// Fast path, works if target is a mount point that can be unmounted.
+// UnmountAll unmounts all mounts and submounts underneath parent,
```

Comment:
```suggestion
// UnmountAll unmounts all mounts underneath parent, excluding the parent itself.
// Note that parent does not have to be a mount point.
```

Reply:
(I've removed "and submount" because from user's perspective all mounts are the same -- it is only in the code we distinguish between the top-level mounts and their sub-mounts).